src/pages/User/components/PullRequests/index.js

```diff
@@ -161,6 +161,22 @@ class PullRequests extends Component {
     return count;
   }
 
+  /**
+   *
+   * @param {*} data
+   * @returns ValidPullRequests.
```

Comment:
Actually this was for the return type. Since JS doesn't have stronger types and your return type is a bit of complex object, you can make it a mixed type `{*}`
```suggestion
   * @returns {*}
```